src/connections/destinations/catalog/algolia-insights/index.md

```diff
@@ -24,31 +24,37 @@ This destination is maintained by [Algolia](https://www.algolia.com/). For any i
 2. Search for "Algolia" in the Catalog, select it, and choose which of your sources to connect the destination to.
 3. Enter the "App ID" & "API Key" into your Segment Settings UI which you can find on the Algolia Dashboard, under API Keys menu.
```

Comment:
> Enter the "App ID" & "API Key" into your Segment Settings UI which you can find on the Algolia Dashboard, under API Keys menu.

Can we show a screenshot of where these items are in the Algolia Dashboard?

Reply:
Yep will add those in today! Makes a lot of sense to show where they're located